cdmtaskservice/jobflows/nersc_jaws.py

```diff
@@ -445,6 +449,35 @@ async def tfunc():
             logpath=os.path.join(self._s3logdir, job.id),
         )
 
+    async def clean_job(self, job: models.AdminJobDetails, force: bool = False):
+        """
+        Clean up job files at the remote compute site.
+        
+        job - the job to clean up.
+        force - perform the clean up even if the job isn't in a terminal state. This may cause
+            undefined behavior.
+        """
+        # might need to have some sort of flowmanager wrapper class that checks these
+        # sort of global issues and ensures that Job is not modified before passing it to the
+        # flow
+        if _not_falsy(job, "job").job_input.cluster != self.CLUSTER:
+            raise ValueError(f"Job cluster must match {self.CLUSTER}")
+        if not force and job.state not in models.JOB_TERMINAL_STATES:
+            raise IllegalParameterError("Job is not in a terminal state and cannot be cleaned")
+        jaws_paths = []
+        if job.jaws_details and job.jaws_details.run_id:
```

Comment:
`run_id` sounds singular, which could be misleading if it refers to multiple runs

Reply:
Right now it's only ever a single run. It's an array to allow for handling retries in the future, but even then, 99% of the time it'll be a single run. It's also been this way for months and the model defines the api and the database structure so I'm going to leave this one as is